config/cfme/manifest_5_11.json

```diff
@@ -24,58 +24,64 @@
       "vms": {
         "id": null,
         "name": null,
+        "description": null,
         "type": null,
-        "guid": null,
+        "ems_ref": null,
```

Comment:
This already exists [here](https://github.com/ManageIQ/topological_inventory-api/pull/224/files#diff-0633ae652eeb9f8a0956d9a21e33c2f9L34)